### ISSUE TITLE
loxberry 3.0.0 and CamStream4Lox Issue - VLC doesn't start

### ISSUE THREAD

**Issue by buchbema** [mschlenstedt/LoxBerry-Plugin-CamStream4Lox#21]
================================================================================
 20.12.2022 20:35:10 TASK STARTED
 INFO: LoxBerry Version 3.0.0 ( is_raspberry.cfg )
 INFO: CamStream4Lox Version 2.0.1
 INFO: Loglevel: 7
 INFO: Starting VLC...
 ERROR: VLC could not be started.
 20.12.2022 20:35:13 TASK FINISHED

**Comment by mschlenstedt** [comment]
VLC hat keinen RTSP Support mehr in Bullseye.

**Comment by buchbema** [comment]
Wird es einen Workaround oder ein anderer Videorenderer implementiert??
Danke für die Antwort

**Comment by mschlenstedt** [comment]
Nein, vermutlich nicht. Außer jemand kann mir sagen, wie ich VLC mit RSTP Unterstützung unter Debian Bullseye manuell installiert bekomme. Leider habe ich das bisher nicht geschafft.

Hilfe ist gerne willkommen: https://www.loxforum.com/forum/projektforen/loxberry/plugins/163516-plugin-camstream4lox-fast-jede-kamera-in-den-türsteuerungsbaustein?p=376295#post376295

Oder auf das MotionEye Plugin umstellen: https://wiki.loxberry.de/plugins/motioneye_plugin/start
